### PR TITLE
feat(organizationinvitation): implement method to list organization invitations from a given instance

### PR DESCRIPTION
**Pull request by NicolasLopes7**

### CODE REVIEW

File: organizationinvitation/client.go [hunk 1]
```diff
@@ -115,3 +115,42 @@ func (c *Client) Revoke(ctx context.Context, params *RevokeParams) (*clerk.Organ
...
+
+	if p.OrderBy != nil {
+		q.Add("order_by", *p.OrderBy)
+	}
+
+	return q
+}
+
+// ListAllFromInstance lists all the organization invitations from the current instance
+func (c *Client) ListFromInstance(ctx context.Context, params *ListFromInstanceParams) (*clerk.OrganizationInvitationList, error) {
```
Comment (gkats) [inline organizationinvitation/client.go:146]: :question: Have we considered reusing the existing `organizationinvitation.List` method?

We could return all organization invitations for an instance if `ListParams.OrganizationID == ""`. Granted, the API would be better if the `OrganizationID` was a pointer to a string, but perhaps a single `List` method reduces the API surface?

What do you think?

Comment (NicolasLopes7) [reply organizationinvitation/client.go:146]: I think the issue here is that we have some additional paramaters that are not contained on the one with organizationID. I can implement it to match both though!

Comment (gkats) [reply organizationinvitation/client.go:146]: > I can implement it to match both though!

Not sure which is better. Feel free to follow your instinct or ask the rest of the team for feedback. :smile:

Comment (NicolasLopes7) [reply organizationinvitation/client.go:146]: I will!

Comment (NicolasLopes7) [reply organizationinvitation/client.go:146]: For now I think it's best to have both

**Review by izaaklauer — APPROVED** [review]

**Pull request merged**